packages/react-components/src/molecules/UsersList.tsx

```diff
@@ -74,27 +74,27 @@ const UsersList: FC<UsersListProps> = ({
   <ul css={listStyles}>
     {users.slice(0, max).map((user, i) => {
       const link = user.id && network({}).users({}).user({ userId: user.id }).$;
+      const internalUser = isInternalUser(user);
+      const imageUrl = internalUser
+        ? user.avatarUrl
+        : getPlaceholderAvatarUrl();
       return (
         <li key={`author-${i}`} css={itemStyles}>
-          {isInternalUser(user) ? (
+          {internalUser ? (
             <div css={userStyles}>
               <ImageLink link={link}>
-                <Avatar {...user} imageUrl={user.avatarUrl} />
+                <Avatar {...user} imageUrl={imageUrl} />
               </ImageLink>
-              <span css={nameStyles}>{user.displayName}</span>
+              <Link ellipsed href={link}>
+                <span css={nameStyles}>{user.displayName}</span>
+              </Link>
               {user.alumniSinceDate && (
                 <span css={iconStyles}>{alumniBadge}</span>
               )}
             </div>
           ) : (
             <div css={userStyles}>
-              {link ? (
-                <ImageLink link={link}>
-                  <Avatar {...user} imageUrl={getPlaceholderAvatarUrl()} />
-                </ImageLink>
-              ) : (
-                <Avatar {...user} imageUrl={getPlaceholderAvatarUrl()} />
-              )}
+              <Avatar {...user} imageUrl={imageUrl} />
```

Comment:
This change removes the possibility of not internal users clicking in the avatar that it's currently possible, right? Is this intentional?

In dev
![clickable-avatar](https://user-images.githubusercontent.com/16595804/195856704-f46ffa7b-315a-43f6-8d3d-3613b066bfd3.gif)

Reply:
I think its a bug. External users don't have a user profile

Reply:
Exactly. Confirmed with Manoel.